sql_generators/glean_usage/templates/clients_last_seen_joined.query.sql

```diff
@@ -22,7 +22,8 @@ SELECT
   baseline.sample_id,
   baseline.submission_date,
   baseline.normalized_channel,
-  * EXCEPT(submission_date, normalized_channel, client_id, sample_id),
+  * EXCEPT(submission_date, normalized_channel, client_id, sample_id, is_default_browser),
+  baseline.is_default_browser,
```

Comment:
@scholtzan removed the COALESCE since metrics doesn't always have this field. From what I can see in this change every baseline should have is_default_browser:

https://github.com/mozilla/bigquery-etl/commit/62bf21ae2e3d3b0a2d7671244ff7a7c225eaf286#diff-2f3e74648a54870a2af64d17ec5c3275b4d689e8d7e6dff2cc855370118eb518